Library/Formula/osquery.rb

```diff
@@ -3,7 +3,7 @@
 class Osquery < Formula
   homepage "http://osquery.io"
   # pull from git tag to get submodules
-  url "https://github.com/facebook/osquery.git", :tag => "1.4.2"
+  url "https://github.com/facebook/osquery.git", :tag => "1.4.3"
```

Comment:
We've added a requirement for a git commit hash along with a tag; can you add `, :revision => "fd99a7f96a3795f56da100da3e6278863fc364d7"` to this line?